src/ray/protobuf/gcs.proto

```diff
@@ -140,11 +141,16 @@ message ActorTableData {
   string name = 12;
   // Last timestamp that the actor state was updated.
   double timestamp = 13;
-  // The task specification of this actor's creation task.
-  TaskSpec task_spec = 14;
+  // Language the job uses.
+  Language language = 14;
   // Resource mapping ids acquired by the leased worker. This field is only set when this
   // actor already has a leased worker.
   repeated ResourceMapEntry resource_mapping = 15;
+  // The function descriptor of the actor creation task.
+  FunctionDescriptor function_descriptor = 16;
```

Comment:
Can we take out stats to a separate message? Maybe call it ActorStats?

Reply:
Hi @rkooo567 , Which variables do you want to put in ActorStats?

Reply:
I thought we pull actor stats data from the gcs, but it is probably not true. Please ignore this comment.

Reply:
Ok, thanks